### PR TITLE
fixed guild join message

### PR DESCRIPTION
**Pull request by saucyapples**
@Fury-101 @ElijahRus250 fixed i think

### CODE REVIEW

**Comment by Fury-101** [comment]
took u long enough

File: index.js [hunk 1]
```diff
@@ -353,14 +353,20 @@
       }
     }
 
+      var welcomeIndex=0;
```
Comment (Fury-101) [inline index.js:356]: my code

Comment (Fury-101) [reply index.js:356]: SMH

**Comment by ElijahRus250** [comment]
Why not select a random item from an array????? this is dumb and slow

**Comment by Fury-101** [comment]
i think the point was so that you dont get spam filtered (idk ig cuz its @saucyapples pr )

**Comment by saucyapples** [comment]
if its random it wont go through cuz spam filter

**Comment by ElijahRus250** [comment]
then add a random id to the end

**Comment by saucyapples** [comment]
ELIJAH ARE YOU DUMB

**Comment by Fury-101** [comment]
i agree

**Comment by saucyapples** [comment]
LMFAO THE WHOLE POINT OF IT IS TO GET RID OF THE DUMB RANDOM IDS

**Comment by Fury-101** [comment]
but why

**Comment by ElijahRus250** [comment]
This is dumb

**Comment by Fury-101** [comment]
ok 03283c0f0a93e9

**Comment by Fury-101** [comment]
GG

**Comment by ElijahRus250** [comment]
Add more sentences to it and ill merge it

**Comment by ElijahRus250** [comment]
10+

**Comment by ElijahRus250** [comment]
this better work, if it doesnt and i merge it you are being whipped

**Pull request merged**